### DISCUSSION TITLE
MACSQuant Files?

### DISCUSSION THREAD

**Discussion by leslie-g-torres** [whitews/FlowKit#137] [Q&A]
Hi,
Can this tool be used with MACSQuant data files? I'm very new to this tool and am a baby coder as well so I'm not sure if I'm just missing something. I keep getting a value error.
-Leslie

![Capture](https://user-images.githubusercontent.com/114437765/212168878-c6b7fa51-a3cb-4f67-a829-a8c8642da636.PNG)

**Comment by whitews** [comment]
Hi Leslie,

I don't think I've tested it with FCS files exported by MACSQuant. Any chance you could share one to experiment with?

Thanks,
Scott

**Reply by leslie-g-torres** [reply]
Hi Scott,
Would love to, but have to ask for permission first. I will get back to you on that! 

Thank you! 
-Leslie
